wagtail/admin/templates/wagtailadmin/panels/read_only_field_panel.html

```diff
@@ -0,0 +1,14 @@
+{% extends "wagtailadmin/shared/field.html" %}
+{% load wagtailadmin_tags i18n %}
+
+{% block form_field_wrapper %}
+    <div class="w-field w-field--textoutput">
+        <div class="w-field__textoutput" role="textbox" tabindex="0" aria-labelledby="{{ heading_id }}"{% if help_text %} aria-describedby="{{ help_text_id }}"{% endif %} aria-readonly="true">
+            {{ display_value }}
+        </div>
+    </div>
+{% endblock %}
```

Comment:
Yes, I figured showing a comment option next to something that is unlikely to ever change between revisions might be misleading from a review perspective. It's certainly not something I have strongly held views on, though.

Reply:
The help text might be used to provide descriptive information about the read-only value. E.g., "use this order number to cross-reference the PayPal transaction" in an eCommerce application.

The `FieldPanel` should have consistent behavior even when adding different modalities. Since  developers are familiar with `help_text`, there is a somewhat high likelihood  that a use case for a `read_only` field will intersect with one for using `help_text`.

Just my two cents 🙂

Reply:
Thanks for the input both! Have left comments showing by default, and devs can use the `disable_comments` option like normal to disable them.